docs/install_guides/installing-openvino-macos.md

```diff
@@ -1,8 +1,8 @@
-# Install and Configure Intel® Distribution of OpenVINO™ toolkit for macOS {#openvino_docs_install_guides_installing_openvino_macos}
+# Install OpenVINO™ Runtime for macOS from Installer
 
-> **NOTE**: Since the OpenVINO™ 2022.1 release, the following development tools: Model Optimizer, Post-Training Optimization Tool, Model Downloader and other Open Model Zoo tools, Accuracy Checker, and Annotation Converter are not part of the installer. These tools are now only available on [pypi.org](https://pypi.org/project/openvino-dev/).
+> **NOTE**: Since the OpenVINO™ 2022.1 release, the OpenVINO Development Tools: Model Optimizer, Post-Training Optimization Tool, Model Downloader and other Open Model Zoo tools, Accuracy Checker, and Annotation Converter are not part of the installer. These tools are now only available on [pypi.org](https://pypi.org/project/openvino-dev/).
```

Comment:
```suggestion
> **NOTE**: Since the OpenVINO™ 2022.1 release, the OpenVINO Development Tools (Model Optimizer, Post-Training Optimization Tool, Model Downloader and other Open Model Zoo tools, Accuracy Checker, and Annotation Converter) are not part of the installer. These tools are now only available on [pypi.org](https://pypi.org/project/openvino-dev/).
```